.github/instructions/changelog.instructions.md

```diff
@@ -0,0 +1,621 @@
+<!-- markdownlint-disable -->
+---
+applyTo: CHANGELOG.md
+---
+
+# AKS Changelog Validation Guidelines
+
+> **Scope**: File-specific patterns for CHANGELOG.md release notes.  
+> **Repo Standards**: See `.github/copilot-instructions.md` for general conventions and Microsoft Style Guide.
+
+---
+
+## Release Structure Template
+
+Every release section MUST follow this structure:
+
+```markdown
+## Release Notes YYYY-MM-DD
+
+> Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `vYYYYMMDD`.
+
+### Announcements of upcoming changes and retirements
+
+* [Announcement item with link to documentation if it exists](https://learn.microsoft.com/...) description of the announcement or retirement.
+
+### Release notes
+
+#### Kubernetes versions

```

Comment:
Deprecation of versions including LTS and launch of minor versions (GA) should be here.  FYI Going forward we should also call out any 'System enforced 'Upgrade dates' ' or forced upgrade from Community to LTS . LTS EOL to community version probably either here or in announcements as the PRD Dan is driving comes to fruition.